target/linux/ar71xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -138,6 +138,8 @@ case "$FIRMWARE" in
 	archer-c59-v1|\
 	archer-c60-v1)
 		ath10kcal_extract "art" 20480 12064
+		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
```

Comment:
Would you mind removing the erroneous line from 6da4f03f026e0375c5d1462 ("ath10k-firmware: add qca9888 firmware") ?

     +	$(INSTALL_DATA) \
     +		$(PKG_BUILD_DIR)/QCA9888/hw2.0/board-2.bin \
     +		$(1)/lib/firmware/ath10k/QCA9888/hw2.0/board.bin 

If you do that you could in theory add the following to 11-ath10k-caldata in the switch/case code:

    "ath10k/QCA9888/hw2.0/board.bin")
              case $board in
              archer-c59-v1)
                    ath10kcal_extract "art" 20480 12064
                    ;;
    ;;

As I said before, I think that the correct way would be make a board-2.bin from the boardData*bin and make a per-device firmware package just like the ipq-wifi from fa03d441e96eee (but for QCA9888). I'm saying this, because Michał Kazior warned me that pushing bad board data can be physically bad for the hardware too.